terraform/modules/front_door/alarms.tf

```diff
@@ -14,7 +14,7 @@ resource "aws_cloudwatch_metric_alarm" "healthy_hosts_count" {
 
   dimensions = {
     LoadBalancer = aws_lb.this.arn_suffix
-    TargetGroup  = aws_lb_target_group.this.arn_suffix
+    TargetGroup  = var.load_balancer_target_group_arn_suffix
```

Comment:
This must not have existed when the JIRA ticket was written else it would be referenced explicitly there, but it will need pulling out of this module in order to meet the criteria that no shared module references anything from a per-app module.

Reply:
This may already be on your list for later! But it seemed worth commenting on because the variable is new.